src/controllers/userContoller.js

```diff
@@ -0,0 +1,103 @@
+const {
+  getUsers,
+  addUser,
+  getUser,
+  deleteUser,
+  updateUser,
+} = require('../services/users.service');
+
+const getAll = (req, res) => {
+  res.statusCode = 200;
+
+  const users = getUsers();
+
+  res.send(users);
+};
+
+const add = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const rawUser = {
+    name,
+  };
+
+  const user = addUser(rawUser);
+
+  res.statusCode = 201;
+  res.send(user);
+};
+
+const get = (req, res) => {
+  const { id } = req.params;
+
+  if (!id) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const user = getUser(+id);
```

Comment:
Consider adding validation for the `id` parameter to ensure it is a valid number before calling `getUser`. This will help prevent potential errors.